### DISCUSSION TITLE
Welcome to DSC-Samples Discussions!

### DISCUSSION THREAD

**Discussion by SteveL-MSFT** [PowerShell/DSC-Samples#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jose-albino** [comment]
Hello

I am starting the learning road to the DSC v3 with a goal to understand how this can be used also in air gap environments to improve compliance and security 

By checking this repository content I do see that it is well structured and that interesting works was made on that level

But when checking samples I do not find any relevant sample for powershell and it seems that work is stopped on the repository 

Can you please provide feedback if this is an active repository? Is this code and samples will be enriched?
